apps/examples/testcase/ta_tc/ble_manager/utc/utc_blemanager_main.c

```diff
@@ -0,0 +1,1128 @@
+/****************************************************************************
+ *
+ * Copyright 2022 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ble_manager/ble_manager.h>
+#include <semaphore.h>
+#include <errno.h>
+#include "ble_mock.h"
+#include "tc_common.h"
+
+#define UTC_FUNC_SIGNAL              \
+	do {                                 \
+		sem_post(&g_func_sem);        \
+	} while (0)
+
+#define UTC_FUNC_WAIT                \
+	do {                                 \
+		sem_wait(&g_func_sem);        \
+	} while (0)
+
+static sem_t g_func_sem;
+
+#define BEVT_CONNECTED 1
+#define BEVT_DISCONNECTED 2
+#define BEVT_NO_NOTI 3
+#define BEVT_NOTI 4
```

Comment:
Possible to align macros like it's been done for some other files in this repository?
```
#define abcd       1
#define pqr        2
#define lmnopq     3
```

Reply:
Done